packages/static/kairos-overlay-files/files/system/oem/09_openrc_services.yaml

```diff
@@ -111,7 +111,6 @@ stages:
           ╚═╝  ╚═╝╚═╝  ╚═╝╚═╝╚═╝  ╚═╝ ╚═════╝ ╚══════╝
                                                       
 
-          Welcome to kairos (login with user: kairos, password: kairos)
```

Comment:
maybe still worth keeping for livecd? though I don't see why someone would log out of the livecd

Reply:
I think you can't log out of the livecd lol, it restarts the service on exit so you get logged in back

Let me try that

Reply:
nope, I think this only happens in alpine, which makes the service restart but not on normal service. You dont even get a login back, just a dead tty lol